python/graphstorm/run/launch.py

```diff
@@ -324,7 +325,7 @@ def construct_torch_dist_launcher_cmd(
     )
 
 def wrap_dist_remap_command(
```

Comment:
Can we take the chance to add a `Returns` docstring?

e.g.

```
        Returns
        -------
        A string that is the original command list, joined and modified to
        include the provided arguments.
```

We can also annotate the return type as string

```
 preserve_input: bool = False) -> str:
```